CMakeLists.txt

```diff
@@ -132,10 +132,13 @@ if(LINUX)
   set(HAVE_UDEV ${UDEV_FOUND})
   find_package(blkid REQUIRED)
   set(HAVE_BLKID ${BLKID_FOUND})
+  find_package(keyutils REQUIRED)
```

Comment:
this is not quite right. keyutils is only used if we want to talk to kernel (module). and we use this only if we build rbd with krbd support or mount.cephfs. both of them can be disabled before and after this change. you are forcing users to install keyutils even if they don't want to build them just because they are build on Linux.

Reply:
@tchaikov 
Fair enough.
Will put the conditional back.